wordpress/wp-content/plugins/memberful-wp/src/shortcodes.php

```
@@ -60,6 +61,10 @@ function memberful_wp_shortcode_account_link( $atts, $content ) {
   return '<a href="'.memberful_account_url().'" role="account">'.do_shortcode($content).'</a>';
 }
 
+function memberful_wp_shortcode_subscriptions_link( $atts, $content ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$atts'. (undefined)

(UnusedFormalParameter)